multicluster/controllers/multicluster/member/clusterset_controller_test.go

```diff
@@ -59,12 +84,14 @@ func TestMemberClusterDelete(t *testing.T) {
 		t.Errorf("Member ClusterSet Reconciler should handle delete event successfully but got error = %v", err)
 	} else {
 		memberClusterAnnounce := &mcv1alpha1.MemberClusterAnnounce{}
-		err := fakeClient.Get(common.TestCtx, types.NamespacedName{
```

Comment:
@luolanzone : could you check if similar mistakes in other tests?

Reply:
No similar mistakes found, but there are some redundant fields for different controllers, I have removed in this PR https://github.com/antrea-io/antrea/pull/5637